test/caption-stream-test.js

```diff
@@ -0,0 +1,283 @@
+(function(window, muxjs, undefined) {
+  'use strict';
+
+  var captionStream;
+
+  module('Caption Stream', {
+    beforeEach: function() {
+      captionStream = new muxjs.mp2t.CaptionStream();
+    }
+  });
+
+  QUnit.skip('parses SEIs larger than 255 bytes', function() {
```

Comment:
Not quite finished-- I'd still have to fill out a more valid `data` property for the captionStream push a couple lines down.